src/config-wrapper.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const { Request, Response } = require('@adobe/helix-fetch');
+const redirect = require('./RedirectConfig');
+
+const loaders = {
+  redirect,
+};
+
+async function getData(request, ...names) {
+  if (/^application\/x-www-form-urlencoded/.test(request.headers.get('content-type'))) {
```

Comment:
hmm...maybe we should transparently support form POSTs in the adapter ?

Reply:
I'd make it explicit. My idea was to create a `bodyData` (name TBD) wrapper that would take the form data or JSON body from a POST or PUT and stash it in `context.data` (TBD)